HHS/Head-Start-TTADP
# [TTAHUB-1120] fix for spacing and display accordion

## Description of change
This fix handles the spacing issue relayed in the ticket (although it looked as though it had been partially fixed already), as well as fixing a related bug where the wrong form field was accessed by the ReviewItem component

## How to test
View the review page and confirm  that the spacing is correct and that the goals & objectives section is populated with data, even if you've opened a goal for editing.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1120


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
